codec/codec_test.go

```diff
@@ -1411,6 +1411,46 @@ func doTestEncUnknownFields(t *testing.T, h Handle) {
 	}
 }
 
+func doTestOmitEmpty(t *testing.T, h Handle) {
+	type O1 struct {
```

Comment:
I'd prefer not starting var names with a capital `O`.  Obviously it's not ambiguous since var names can't start with numbers, but I still think using `S` or something would be easier to read.